chocolatey/copenhagen_theme
# (#28) Upgrade to choco-theme 0.8.1

## Description Of Changes
This upgrades this repository to choco-theme 0.8.1. For more information about this release see:
https://github.com/chocolatey/choco-theme/releases/tag/0.8.1.

Read all the release notes for choco-theme at:
https://github.com/chocolatey/choco-theme/releases

## Motivation and Context
We should be using the latest versions of this package to remove any potential security vulnerabilities and to take advantage of new features and fixes.

## Testing
1. Install zcli with `npm install @zendesk/zcli -g`
2. From the root of this PR, run `zcli login -i`
3. Once authenticated, run `zcli themes:preview ` to view the site.
4. Click around, and ensure all looks good. Change the theme modes to ensure no JS errors, and the them color switches.

### Operating Systems Testing
Windows 10

## Change Types Made

## Change Checklist


## Related Issue
* ENGTASKS-4000
* #28 
* https://github.com/chocolatey/img/pull/15
